components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -239,6 +269,30 @@ const bankAccountMutation = gqlV2`
   }
 `;
 
+const applyToHostMutation = gqlV2`
+mutation applyToHost($collective: AccountReferenceInput!, $host: AccountReferenceInput!) {
+  applyToHost(collective: $collective, host: $host) {
+    id
+    slug
+    host {
+      id
+      slug
+    }
+  }
+}
+`;
+
+const addApplyToHostMutation = graphql(applyToHostMutation, {
+  options: {
+    context: API_V2_CONTEXT,
+  },
+  props: ({ mutate }) => ({
```

Comment:
I'm now recommending to avoid this syntax and just use `name`. See what I did in https://github.com/opencollective/opencollective-frontend/pull/4027 for example. Don't forget to add `variables` when using it.